server/config.go

```diff
@@ -408,6 +425,36 @@ func parseClusteringConfig(config *Config, m map[string]interface{}) error { //
 	return nil
 }
 
+// parseActivityStreamConfig parses the `activitystream` section of a config
+// file and populates the given Config.
+func parseActivityStreamConfig(config *Config, m map[string]interface{}) error { // nolint: gocyclo
```

Comment:
Is it a decision that the activity stream is never partitioned?

Reply:
No, we could (and should) definitely add an option to enable partitions for this stream. I'll add that.

Reply:
Should be fixed in 47cc4d7.